ansible/roles/etcd/tasks/etcd2-configure.yml

```diff
@@ -11,10 +11,3 @@
 - name: CoreOS | Write etcd config file
   template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
   notify: restart etcd2
```

Comment:
@ingvagabund I see your point. I guess the fix would be just to have `start.yml` called before `configure.yml` [here] (https://github.com/stephenrlouie/contrib/blob/etcd-ref/ansible/roles/etcd/tasks/main.yml#L4-L6)

I believe this would resolve the `etcd2.started` and `etcd.started` handler issue. But feels sloppy to me.

This should work as is since the configuration is put in place before we start it, so it shouldn't require a restart. A more pressing concern would be when you want to run a reconfigure play or an upgrade play. You need to make sure that handler gets called properly. I'm not sure what would be optimal, keep that condition on the handler and make sure its called on a reconfigure or upgrade, or remove that condition and always restart when that handler is called?

Advice on the best choice would be appreciated @danehans @eparis @ingvagabund